libs/minecraft/PosUtils.arucas

```diff
@@ -136,4 +179,24 @@ enum Direction {
         }
         return Direction.NONE;
     }
+    static fun getAllDirections(){
+        return Direction.ALL;
+    }
+
+    static fun getHorizontalDirections(){
+        return Direction.HORIZONTAL;
+    }
+
+    static fun getVerticalDirections(){
+        return Direction.VERTICAL;
+    }
+
+    static fun getDirectionsFromAxis(string){
+            switch(string.lowercase()) {
+                case "x" -> return Axis.AXIS_X;
```

Comment:
Name doesn't fit the function `getAxisFromString` (also probably should be moved into the `axis` enum.